rendering.php

```diff
@@ -849,6 +849,15 @@ public function renderNoteText($note) {
 
             $text .= ($publication != '') ? ',' : '.';
         }
+        elseif ($data->has('booktitle')) {  // rwduzhao
```

Comment:
Please don't leave vanity comments. You can add extra author in the file header if you really want to.